.github/workflows/build-and-test.yaml

```diff
@@ -0,0 +1,10 @@
+name: Build and Test
```

Comment:
We already have a build and test pipeline in `node.js.yaml` - would this be better added to that? Otherwise we'll be triggering two separate build and test runs on each PR.

Reply:
Sure I can consolidate the two